crates/autopilot/src/driver_model.rs

```diff
@@ -85,6 +85,8 @@ pub mod execute {
     #[serde(rename_all = "camelCase")]
     pub struct Request {
         pub auction_id: i64,
+        #[serde(with = "bytes_hex")]
+        pub tag: Vec<u8>,
```

Comment:
Why are you calling it `tag` when I called it `id`? Note that this "tag" would also be used to identify the transaction for `/execute` as well. I still feel like `id` is a better name, or `solutionId` if you want to be wordy.

Reply:
Mistake in this PR. It is called `transactionIdentifier` in the api docs so I will name it that here too. Do you think it should be named something else in the api?